src/main/java/edu/ucar/unidata/rosetta/service/FileParserManagerImpl.java

```diff
@@ -78,42 +85,29 @@ public String parseByLine(String filePath) {
      * used to parse the file data when both the header lines and delimiter(s)
      * have been specified by the user. TODO: refactor to return JSON
      *
-     * @param filePath          The path to the file on disk.
-     * @param selectedDelimiter The delimiter selected to which any other delimiters will be
-     *                          normalized.
-     * @param delimiterList     The List<String> of delimiters specified by the user.
-     * @param headerLineList    The List<String> of header lines specified by the user.
+     * @param filePath   The path to the file on disk.
+     * @param delimiter  The used delimiter to parse the data.
+     * @param headerLineList  The List<String> of header lines specified by the user.
      * @return A String of the file data parsed by the delimiter(s).
      */
-    public String normalizeDelimiters(String filePath, String selectedDelimiter, List<String> delimiterList, List<String> headerLineList) {
-        List<List<String>> parsedData = new ArrayList<List<String>>();
+    /*
+    public String normalizeDelimiters(String filePath, String delimiter, List<String> headerLineList) throws IOException {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)